packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -56,6 +56,9 @@ type TooltipProps = {
   delay?: number | [number | null, number | null];
 };
 
+type Plugins = NonNullable<React.ComponentProps<typeof Tippy>["plugins"]>;
+type Plugin = Plugins[0];
```

Comment:
`@tippyjs/react` does not expose `tippy.js` types, have to extract via props and grab element type from array type

Reply:
ooh fancy -- would `Plugins[number]` work here? (theoretically should generate a union of the possible element types, and is a bit more general)

also, let's leave your comment as a code comment 🙂

Reply:
Will do, and also changed to `number`